File: Csharp/Structural/Bridge/Show/Show/Radio.cs
namespace Show
{
    public class Radio: Device
    {
        private bool on = false;
        private int volume = 10;

        public bool IsEnabled() => on;

        public void Enable() => on = true;

        public void Disable() => on = false;

        public int GetVolume() => volume;

        public void SetVolume(int percent) => volume = percent;
    }
}

File: Python/Structural/Bridge/Example2/Show/src/Radio.java
public class Radio implements Device {
    private int volume;
    @Override
    public void turnOn() {
        System.out.println("TV turned on.");
    }

    @Override
    public void turnOff() {
        System.out.println("TV turned off.");
    }

    @Override
    public void setVolume(int percent) {
        this.volume = percent;
        System.out.println("TV volume set to " + this.volume + "%");
    }
}
